adapters/handlers/rest/configure_api.go

```diff
@@ -956,6 +957,14 @@ func registerModules(appState *state.State) error {
 			Debug("enabled module")
 	}
 
+	if _, ok := enabledModules[modmulti2vecvoyageai.Name]; ok {
+		appState.Modules.Register(modmulti2vecvoyageai.New())
+		appState.Logger.
+			WithField("action", "startup").
+			WithField("module", modmulti2vecvoyageai.Name).
+			Debug("enabled module")
+	}
```

Comment:
You want to add it to the default modules?

Reply:
the default modules setting is available from `v1.26` I believe, I will not forget to add it during merge